sdk/src/CrashReporter.ts

```diff
@@ -321,9 +297,13 @@ export default class CrashReporter {
         : payload;
 
     if (!modifiedPayload) {
+      RaygunLogger.i("Payload ignore from beforeSendHandler");
```

Comment:
This log I think can be removed because the customer is in control of this action and can log the event themselves in their on before send method